diff --git a/node_modules/swiper/modules/thumbs/thumbs.js b/node_modules/swiper/modules/thumbs/thumbs.js
index 80c2b80..b98cd91 100644
--- a/node_modules/swiper/modules/thumbs/thumbs.js
+++ b/node_modules/swiper/modules/thumbs/thumbs.js
@@ -24,7 +24,7 @@ export default function Thumb({
     if (!thumbsSwiper || thumbsSwiper.destroyed) return;
     const clickedIndex = thumbsSwiper.clickedIndex;
     const clickedSlide = thumbsSwiper.clickedSlide;
-    if (clickedSlide && clickedSlide.classList.contains(swiper.params.thumbs.slideThumbActiveClass)) return;
+    if (clickedSlide && clickedSlide?.classList?.contains?.(swiper.params.thumbs.slideThumbActiveClass)) return;
     if (typeof clickedIndex === 'undefined' || clickedIndex === null) return;
     let slideToIndex;
     if (thumbsSwiper.params.loop) {
@@ -65,7 +65,7 @@ export default function Thumb({
       swiper.thumbs.swiper = new SwiperClass(thumbsSwiperParams);
       swiperCreated = true;
     }
-    swiper.thumbs.swiper.el.classList.add(swiper.params.thumbs.thumbsContainerClass);
+    swiper.thumbs.swiper.el?.classList?.add?.(swiper.params.thumbs.thumbsContainerClass);
     swiper.thumbs.swiper.on('tap', onThumbClick);
     return true;
   }
@@ -84,17 +84,17 @@ export default function Thumb({
       thumbsToActivate = 1;
     }
     thumbsToActivate = Math.floor(thumbsToActivate);
-    thumbsSwiper.slides.forEach(slideEl => slideEl.classList.remove(thumbActiveClass));
+    thumbsSwiper.slides.forEach(slideEl => slideEl?.classList?.remove?.(thumbActiveClass));
     if (thumbsSwiper.params.loop || thumbsSwiper.params.virtual && thumbsSwiper.params.virtual.enabled) {
       for (let i = 0; i < thumbsToActivate; i += 1) {
         elementChildren(thumbsSwiper.slidesEl, `[data-swiper-slide-index="${swiper.realIndex + i}"]`).forEach(slideEl => {
-          slideEl.classList.add(thumbActiveClass);
+          slideEl?.classList?.add?.(thumbActiveClass);
         });
       }
     } else {
       for (let i = 0; i < thumbsToActivate; i += 1) {
         if (thumbsSwiper.slides[swiper.realIndex + i]) {
-          thumbsSwiper.slides[swiper.realIndex + i].classList.add(thumbActiveClass);
+          thumbsSwiper.slides[swiper.realIndex + i]?.classList.add?.(thumbActiveClass);
         }
       }
     }